data_labeling_examples/bulk_labeling_java/src/main/java/com/oracle/datalabelingservicesamples/scripts/RemoveLabelScript.java

```diff
@@ -0,0 +1,183 @@
+                log.error("No Label found with prefix {}",removeLabelPrefix );
+                throw new InvalidParameterException("Invalid Input RemoveLabelPrefix Provided");
+            }
+        }
+        log.info("Labels to be removed");
+        log.info(String.valueOf(labelsToRemove));
+        String page = null;
+
+        do{
+            ListAnnotationsRequest listAnnotationsRequest = ListAnnotationsRequest.builder().compartmentId(dataset.getCompartmentId()).datasetId(datasetId).lifecycleState(Annotation.LifecycleState.Active).page(page).build();
+            ListAnnotationsResponse response = Config.INSTANCE.getDlsDpClient().listAnnotations(listAnnotationsRequest);
+            assert response.get__httpStatusCode__() == 200 : "List Annotation Response Fail";
+
+            if(response.getAnnotationCollection().getItems().size()>0){
+                List<CompletableFuture<Void>> completableFutures = new ArrayList<>();
+                for(AnnotationSummary annotation : response.getAnnotationCollection().getItems()){
+                    CompletableFuture<Void> future = CompletableFuture
+                            .runAsync(() -> processAnnotationSummaryAndRemoveLabel(annotation),executorService);
+                    completableFutures.add(future);
+                }
+                CompletableFuture<Void> combinedFuture = CompletableFuture
+                        .allOf(completableFutures.toArray(new CompletableFuture[0]));
+                combinedFuture.get();
+            }
+            page = response.getOpcNextPage();
+        }while(page != null);
+        executorService.shutdown();
+        log.info("Total annotations to process : {}, "
+                        +"Successfully processed {}, "
+                        +"Failed to process {}",
+                successCount.get() + failureCount.get(),
+                successCount.get(),
+                failureCount.get());
+        long elapsedTime = System.nanoTime() - startTime;
+        log.info("Time Taken is {} second(s)", elapsedTime / 1_000_000_000);
+    }
+
+    private static void validateRequest(String datasetId) {
+        /*
+         * Validate Dataset
+         */
+        GetDatasetRequest getDatasetRequest = GetDatasetRequest.builder().datasetId(datasetId).build();
+        GetDatasetResponse datasetResponse = Config.INSTANCE.getDlsDpClient().getDataset(getDatasetRequest);
+        assert datasetResponse.get__httpStatusCode__() == 200 : "Invalid Dataset Id Provided";
+        List<LabelName> datasetLabelSet = datasetResponse.getDataset().getLabelSet().getItems();
+
+        log.info("Dataset Labels : ");
+        for (LabelName labelName : datasetLabelSet) {
+            actualDatasetLabels.add(labelName.getName());
+            log.info(labelName.getName());
+        }
+        dataset = datasetResponse.getDataset();
+    }
+    private static void processAnnotationSummaryAndRemoveLabel(AnnotationSummary annotationSummary){
```

Comment:
Requires assertions to prevent NPE during get calls and exception handling

Reply:
AnnotationSummary can't be null. 
And after fetching the annotation details using getAnnotation API, we have added the assertion for API response.